modules/sandboxed-containers-preparing-openshift-cluster.adoc

```diff
@@ -8,14 +8,24 @@
 
 Before you install {sandboxed-containers-first}, ensure that your {product-title} cluster meets the following requirements:
 
-* Your cluster must be installed on bare-metal infrastructure, on premise with {op-system-first} workers.
+* Your cluster must be installed on on-premise bare metal infrastructure with {op-system-first} workers. You can use any installation method including user-provisioned, installer-provisioned, or assisted installer to deploy your cluster.
```

Comment:
For adjective, hyphenate "bare-metal". RH SG.

Also, looks like there may be two spaces before the second sentence.